### PR TITLE
Update About page text, contributors, and organization

### PR DESCRIPTION
**Pull request by russtuck**

### CODE REVIEW

File: src/views/About/contributors.json [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
+    "title": "Summer Practicum, 2021",
+    "body": "Cameron Abbot '23, David Gurge '23, Hyungyu Park '22, Joshua Peters '23, Kaiwen Chu '23, Roddy Ngolomingi '22, Silas White '23 (Directors: Jonathan Senning, Ph.D. '85 and Russ Tuck, Ph.D.)"
+  },
+  {
+    "title": "GoCo Tech Labs, 2021-22",
+    "body": "Bethany Ruggerio '23, Cameron Abbot '23, Kaiwen Chu '23, Hyungyu Park '22, Silas White '23 (Directors: Evan Platzer '20, Bennett Forkner, Information Systems Group)"
+  },
+  {
+    "title": "GoCo Tech Labs, 2022-23",
+    "body": " (Directors: Evan Platzer '20, Bennett Forkner, Information Systems Group)"
```
Comment (EjPlatzer) [inline src/views/About/contributors.json:56]:
```suggestion
    "body": "Arabella Ji '24, Charlie Lee '23, Jake Colbert '23, Kaiwen Chu '23, Sharon Patta '24, Sofi Stonehouse '25 (Directors: Evan Platzer '20, Bennett Forkner, Information Systems Group)"
```

File: src/views/About/index.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Fragment, useState, useEffect } from 'react';
 import versionService from 'services/version';
 import { projectName } from 'project-name';
```
Comment (jsenning) [inline src/views/About/index.jsx:3]: This is no longer needed.

**Review by jsenning — CHANGES_REQUESTED** [review]
Thanks for doing this!

File: src/views/About/index.jsx (hunk 2 above)
Comment (russtuck) [reply src/views/About/index.jsx:2]: Thanks.  Done.

**Comment by russtuck** [comment]
Merging from develop broke dark mode for the version info at the bottom of the About page (which was broken before this PR, but worked earlier in it).  That would be nice to fix, but isn't worth holding back the rest of these changes (including fixing the Contact us line just above that).

**Pull request merged**